microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/MultipleJarDeploymentTest.java

```diff
@@ -0,0 +1,99 @@
+        String mpConfig = "hystrix.command.default.execution.timeout.enabled=true";
+
+        return ShrinkWrap.create(JavaArchive.class, FIRST_DEPLOYMENT_JAR + ".jar")
+                .addClasses(HelloService.class, MyContext.class, HelloFallback.class, FallbackHandler.class)
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(new StringAsset(mpConfig), "microprofile-config.properties");
+    }
+
+    @Deployment(name = SECOND_DEPLOYMENT_JAR, order = 2)
+    public static Archive<?> createSecondJarDeployment() {
+        String mpConfig = "hystrix.command.default.execution.timeout.enabled=false";
+
+        return ShrinkWrap.create(JavaArchive.class, SECOND_DEPLOYMENT_JAR + ".jar")
+                .addClasses(HelloService.class, MyContext.class, HelloFallback.class, FallbackHandler.class)
+                .addClasses(MultipleJarDeploymentTest.class, LogChecker.class)
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(new StringAsset(mpConfig), "microprofile-config.properties");
+    }
+
+    /**
+     * @tpTestDetails Deploy first and then second MP FT application. Both of them are the same (same classes/methods)
+     * @tpPassCrit Verify that second deployment causes that there is logged warning: "WFLYMPFTEXT0002: Hystrix was already
+     *             configured!
+     *             ..." and there is no ERROR message.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @RunAsClient
+    public void testDeploySecondJarLogsHystrixNoConfigChangeWarning() throws Exception {
+        try (final OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            final LogChecker logChecker = new ModelNodeLogChecker(client, 10, true);
+
+            Assert.assertTrue(
+                    "Log does not contain WARNING Log: WFLYMPFTEXT0002: Hystrix was already configured! Skipping configuration from deployment - but it should.",
+                    logChecker.logMatches(Pattern.compile(".*WFLYMPFTEXT0002.*")));
+            Assert.assertFalse("Log must not contain any ERROR logs but there are. See server log." + logChecker.toString(),
+                    logChecker.logMatches(Pattern.compile(".*ERROR.*")));
+        }
+    }
+
+    /**
+     * @tpTestDetails Deploy first and then second MP FT application. Verify they're working.
+     * @tpPassCrit Verify that second deployment does NOT change MP FT/Hystrix configuration by disabling @Timeout
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @OperateOnDeployment(SECOND_DEPLOYMENT_JAR) // enrich this deployment by test
+    public void testFaultToleranceTimeoutOnSecondDeployment() throws Exception {
+        MatcherAssert.assertThat("Microprofile Fault Tolerance does not work on 2nd deployment.",
+                helloService.timeout(true), CoreMatchers.containsString("Fallback Hello, context = foobar"));
```

Comment:
On other places you are using statically imported `org.hamcrest.Matchers.containsString`. But here `import org.hamcrest.CoreMatchers.containsString`. Why?

Reply:
thanks, fixed